packages/core/src/layout/FlexLayout/FlexLayout.tsx

```diff
@@ -57,7 +57,7 @@ export const FlexLayout = forwardRef<HTMLDivElement, FlexLayoutProps>(
       justify,
       separators,
       style,
-      wrap,
+      wrap = true,
```

Comment:
Should this be kept as wrap or changed to disableWrap?

Reply:
I think `disableWrap` makes sense, I've amended the prop